scripts/library.sh

```diff
@@ -652,6 +652,21 @@ function get_canonical_path() {
   echo "$(cd ${path%/*} && echo $PWD/${path##*/})"
 }
 
+# List changed files in the current PR.
+# This is implemented as a function so it can be mocked in unit tests.
+# It will fail if a file ever contained a newline character (which is bad practice anyway)
```

Comment:
This command will fail if the file name contains a newline character, or the file content? It would be surprising if it's the latter..

Reply:
fileNAME yes, I can update the comment to include the word `name`